src/gnark_backend_wrapper/groth16/mod.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
In the case of this module, we'll just want this allow for the function and not for the entire module. The case of `acvm_interop` was a particular one.